data/json/items/comestibles/med.json

```diff
@@ -1437,5 +1437,23 @@
       "move_cost": 100,
       "effects": [ { "id": "pkill1", "duration": 720 } ]
     }
+  },
+  {
+    "id": "strong_antibiotic",
+    "type": "COMESTIBLE",
+    "comestible_type": "MED",
+    "name": "trioxycycline",
+    "name_plural": "trioxycyclines",
```

Comment:
Redundant.

Reply:
I was intending to name it something non-existent, like Atreyupan. I'll remove the plural field though.